pom.xml

```diff
@@ -168,7 +168,7 @@
     <rocksdbjni.version>7.5.3</rocksdbjni.version>
     <spark33.version>3.3.4</spark33.version>
     <spark34.version>3.4.3</spark34.version>
-    <spark35.version>3.5.4</spark35.version>
+    <spark35.version>3.5.5</spark35.version>
```

Comment:
@CTTY should we consider upgrading the Spark versions in the GH CI as well, including bundle validation?